.circleci/config.yml

```diff
@@ -143,7 +143,7 @@ jobs:
           command: |
             pushd .
             cd /tmp
-            curl -L "https://releases.hashicorp.com/terraform/1.6.2/terraform_1.6.2_linux_amd64.zip" -o terraform.zip
+            curl -L "https://releases.hashicorp.com/terraform/1.5.7/terraform_1.5.7_linux_amd64.zip" -o terraform.zip
```

Comment:
Terraform 1.5.x is already included in circleci image, so to revert back we just need to remove installation steps

https://github.com/gruntwork-io/terragrunt/blob/master/.circleci/config.yml#L14